test/new-e2e/tests/updater/linux_test.go

```diff
@@ -30,7 +30,7 @@ func TestUpdaterSuite(t *testing.T) {
 
 func (v *vmUpdaterSuite) TestUpdaterOwnership() {
 	// updater user exists and is a system user
-	require.Equal(v.T(), "/usr/sbin/nologin\n", v.Env().RemoteHost.MustExecute(`getent passwd dd-updater | cut -d: -f7`))
+	require.Equal(v.T(), "/usr/sbin/nologin\n", v.Env().RemoteHost.MustExecute(`getent passwd dd-updater | cut -d: -f7`), "unexpected: user does not exist or is not a system user")
 	// owner
 	require.Equal(v.T(), "dd-updater\n", v.Env().RemoteHost.MustExecute(`stat -c "%U" /opt/datadog/updater`))
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
You might add logs to other assertion too